app/features/app/components/App.js

```diff
@@ -50,3 +50,5 @@ export default class App extends Component<*> {
         );
     }
 }
+
+export default App;
```

Comment:
Can you please undo this change altogether? It's just not needed so it only adds "nose" to the PR.